### PR TITLE
Add id verification to bypass ratelimiting

### PR DESCRIPTION
**Pull request by fixe**

### CODE REVIEW

File: test/ratelimit.js [hunk 1]
```diff
@@ -63,49 +63,42 @@ describe('ratelimit middleware', function() {
...
-    });
+      app.use(ratelimit({
+        db: db,
+        id: function (ctx) {
+          return false;
+        },
+        max: 5
+      }));
 
-    it('should not limit when different ids', function (done) {
```
Comment (fixe) [inline test/ratelimit.js]: This case is already covered by `should not limit if id returns `false`.

Comment (dead-horse) [inline test/ratelimit.js]: `should not limit if id returns false` didn't cover ids are different, it just covered `id === false`.  i think we should better to add a test case for this situation.

Comment (fixe) [inline test/ratelimit.js]: I think that the test you mention is the responsibility of the `ratelimiter` package. Do you still want me to add an integration test?

Comment (dead-horse) [inline test/ratelimit.js]: yep, this is also a feature of `koa-ratelimit`, so i think we'd better add this test case.

**Comment by fixe** [comment]
@dead-horse rebased and added missing test.

**Comment by dead-horse** [comment]
:+1:

**Pull request merged**